indico/modules/events/abstracts/forms.py

```diff
@@ -200,6 +227,34 @@ def __init__(self, *args, **kwargs):
         self.merged_into.excluded_abstract_ids = {abstract.id}
 
 
+class AbstractReviewForm(IndicoForm):
+    """Form for reviewing an abstract"""
+
+    _order = ('comment', 'proposed_action', 'proposed_contribution_type', 'proposed_related_abstract')
+
+    comment = TextAreaField(_("Comments"))
+    proposed_action = IndicoEnumSelectField(
+        _("Proposed Action"), [DataRequired()],
+        enum=AbstractAction, skip={AbstractAction.change_tracks})
+    proposed_related_abstract = AbstractField(
+        _("Target Abstract"),
+        [HiddenUnless('proposed_action', {AbstractAction.mark_as_duplicate, AbstractAction.merge}), DataRequired()],
+        description=_("The abstract should be marked as duplicate of the selected one"),
```

Comment:
For consistency, maybe. We refer as "The current abstract" in `AbstractJudgmentFormBase` descriptions.